core/store/migrations/0040_add_run_id_to_pipeline_runs.go

```diff
@@ -0,0 +1,42 @@
+package migrations
+
+import (
+	"gorm.io/gorm"
+)
+
+const up40 = `
+	CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+	ALTER TABLE pipeline_task_runs ADD COLUMN run_id uuid NOT NULL UNIQUE DEFAULT uuid_generate_v4 ();
+	ALTER TABLE pipeline_runs ADD COLUMN inputs jsonb;
+
+	CREATE TYPE pipeline_runs_state AS ENUM (
+	    'unknown',
```

Comment:
Seems weird to encode "unknown" as an actual enum type... what does it mean?

Reply:
Agreed, I'll drop this